docs/howtoguides/how_to_know_if_system_affected_by_cve.rst

```diff
@@ -0,0 +1,185 @@
+.. _pro-fix-check-cve:
+
+How to Determine if Your System is Affected by a CVE/USN?
+*********************************************************
+
+.. include:: ../includes/pro-fix-intro.txt
+
+
+If you've come across a Common Vulnerabilities and Exposure (CVE) or a Ubuntu Security Notification (USN) and want to check if your Ubuntu system is vulnerable, you can easily check using the `pro fix` command. In this guide we'll show you how to determine if your system is affected by a specific CVE:
+
+.. note::
+    The `pro fix` command is part of the `pro` package, which is a security tool for Ubuntu systems. If you don't have it installed, you can install it by following the instructions in the `pro` package documentation.
```

Comment:
```suggestion
    The ``pro fix`` command is part of the Ubuntu Pro Client (``pro``), which is a security tool for Ubuntu systems. If you don't have it installed, you can install it by following the instructions in the ``pro`` package documentation.
```

Reply:
Same comments as in previous page, we'll want to be helpful by either giving them the link they need, or providing the code.